pkg/agent/agent.go

```diff
@@ -730,6 +729,9 @@ func (i *Initializer) getNodeMTU(localIntf *net.Interface) (int, error) {
 	if i.networkConfig.EnableIPSecTunnel {
 		mtu -= config.IpsecESPOverhead
 	}
+	if i.nodeConfig.PodIPv6CIDR != nil {
```

Comment:
I am thinking if IPv6 extra cost is in the tunnel header, shall we check the address family of Node's Internal Address instead of the PodCIDR's?

Reply:
Make sense. Let's created a new PR to fix this.